app/src/main/java/org/fossasia/openevent/general/event/EventsFragment.kt

```diff
@@ -31,33 +29,21 @@ class EventsFragment : Fragment() {
         rootView.eventsRecycler.adapter = eventsRecyclerAdapter
         rootView.eventsRecycler.isNestedScrollingEnabled = false
 
-        linearLayoutManager = LinearLayoutManager(context)
-        rootView.eventsRecycler.layoutManager = linearLayoutManager
```

Comment:
Why no layout manager

Reply:
It is there look at line 27 ` rootView.eventsRecycler.layoutManager = LinearLayoutManager(activity)`